### DISCUSSION TITLE
[Feat Request] A clickable link on the Radio current music/band playing.

### DISCUSSION THREAD

**Discussion by edersong** [music-assistant/support#2111] [Feature requests and ideas]
A clickable link on the Radio's current music playing where lead us directly to the search bar with that music/band on it to add it to a Playlist.
Currently, we need to open the Shazam or Google to find the music, go to music provider to add the music to a playlist. It will simplify a lot it.

**Comment by OzGav** [comment]
That would rely on the radio station sending useful metadata but often they do not.

**Comment by edersong** [comment]
Sure, But for the Radios that send that information it will be so useful

**Comment by OzGav** [comment]
It’s not good UX for it to work sometimes and not others. Have a look at the attributes for the player and you might find it there as I think Marcel added the info to that if it was available. You could use that in your own automation if so.

**Comment by edersong** [comment]
Thank you for the idea, but I'm using the docker container version, not HA version.
For the UX, when the name is not provided, it will not be a problem because there is nothing to be clicked.

**Comment by OzGav** [comment]
This has been implemented in MA 2.4

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
